src/hooks/useConnectCurrentEvent.ts

```diff
@@ -1,16 +1,28 @@
 import { useParams } from "react-router-dom";
 import { useFirestoreConnect } from "react-redux-firebase";
+import { oneHourAfterTimestamp } from "utils/time";
+import { useState, useEffect } from "react";
 import { useUser } from "./useUser";
 
 const useConnectCurrentEvent = () => {
-  const { venueId, eventId } = useParams();
+  const { venueId } = useParams();
   const { user } = useUser();
+  const [currentTimestamp, setCurrentTimestamp] = useState(0);
+
+  useEffect(() => {
+    setCurrentTimestamp(Date.now() / 1000);
+  }, []);
 
   useFirestoreConnect([
     {
       collection: "venues",
       doc: venueId,
-      subcollections: [{ collection: "events", doc: eventId }],
+      subcollections: [{ collection: "events" }],
+      where: [
+        ["start_utc_seconds", ">=", oneHourAfterTimestamp(currentTimestamp)],
```

Comment:
neat. this type of where clause and limiting would really help make the party map load faster during parties; I think we load every user in firestore on every request right now :/